### DISCUSSION TITLE
Universal Calculator

### DISCUSSION THREAD

**Discussion by JessicaOrj** [JessicaOrj/Tafe240267#3] [General]
Hello
I have finished creating the basic features for our project. What is the name of the branch we will have to use for the individual development work ?

**Comment by ghost** [comment]
Hello Jessica,

Thank you for setting up the basic features. For individual development work, we should each create a branch of the main branch with the naming Feature_Mortgage_Calculator. This will help keep our work organized and make it easier to merge later.

Let me know if that works for you!
